src/app/segmentdetails/segment-features.component.html

```diff
@@ -49,5 +49,17 @@ <h4 mat-line><strong>OCR</strong></h4>
                 {{ocr}}
             </div>
         </mat-tab>
+        <mat-tab *ngIf="_meta.length>0" label="Metadata">
+
+            <table mat-table [dataSource]="_meta">
+                <ng-container matColumnDef="{{meta}}" *ngFor="let meta of metaCols">
```

Comment:
I guess the only missing part now is filtering / searching for  specific meta data